### PR TITLE
Updated .NET samples

### PR DESCRIPTION
**Pull request by geel9**
- Updated .NET Samples
	- Does not include `SearchRegistry` sample -- @tmarkovski please advise (current samples don't include an example query)
- Uses following baseline parameters:
	- Ecosystem name: `my-ecosystem`
	- Template:
		- Name: `An Example Credential` / `an-example-credential` / `urn:template:my-ecosystem:an-example-credential`
		- Fields: `firstName`, `lastName`, `age`
	- Wallet email: `jane.doe@example.com`
	- Credential ID: `urn:uuid:7afd0191-9656-3a51-c12f-44ecfbb5a829`
	- Revocation ID: `urn:revocation-registry:my-ecosystem:AD9wKZCr4zVxat1XSawScD#42`
	- Trust Registry ID: `urn:egf:my-ecosystem:1297e60abb3b425faaaf0f0a551aae2c`

### CODE REVIEW

File: docs/reference/services/provider-service.md [hunk 1]
```diff
@@ -123,26 +87,6 @@ Returns the public key being used to create/verify oberon tokens
...
-## RefreshDomainVerificationStatus
-
-Call to verify domain
-
-{{ proto_sample_start("services.provider.v1.Provider.RefreshDomainVerificationStatus") }}
-{{ proto_sample_code("services.provider.v1.Provider.RefreshDomainVerificationStatus") }}
-{{ proto_method_tabs("services.provider.v1.Provider.RefreshDomainVerificationStatus") }}
-
----
-
```
Comment (fundthmcalculus) [inline docs/reference/services/provider-service.md]: Why are these removed?

File: docs/reference/services/provider-service.md [hunk 2]
```diff
@@ -76,42 +76,6 @@ Presently, once a wallet's DID has been upgraded, its DID Method cannot be chang
...
-## InvitationStatus
-
-This will be removed May 1, 2023
-Check the status of an invitation
-
-{{ proto_sample_start("services.provider.v1.Provider.InvitationStatus") }}
-{{ proto_sample_code("services.provider.v1.Provider.InvitationStatus") }}
-{{ proto_method_tabs("services.provider.v1.Provider.InvitationStatus") }}
-
----
```
Comment (fundthmcalculus) [inline docs/reference/services/provider-service.md]: Do we want to remove these yet?

**Review by fundthmcalculus — CHANGES_REQUESTED** [review]
Overall, I approve of the code snippets. Let's not conflate the removal of deprecated endpoints from the docs with code snippet creation.

File: docs/reference/services/provider-service.md (hunk 1 above)
Comment (geel9) [reply docs/reference/services/provider-service.md]: My understanding is we had agreed at some point that these should not be part of the public-facing SDK but rather the job of the Dashboard -- I can add samples back in if we like but I'd prefer to move forward with porting them out of the SDK.

File: docs/reference/services/provider-service.md (hunk 2 above)
Comment (geel9) [reply docs/reference/services/provider-service.md]: These haven't been called a single time in the past month and we plan to remove them shortly -- might as well bite the bullet and not perform the work of documenting endpoints that are effectively already dead.

**Comment by geel9** [comment]
> Overall, I approve of the code snippets. Let's not conflate the removal of deprecated endpoints from the docs with code snippet creation.

Fair -- I'll make a separate PR and the discussion can live there

File: docs/reference/services/provider-service.md (hunk 1 above)
Comment (fundthmcalculus) [reply docs/reference/services/provider-service.md]: Agreed. Let's remove these.

File: docs/reference/services/provider-service.md (hunk 2 above)
Comment (fundthmcalculus) [reply docs/reference/services/provider-service.md]: @janpieterz thoughts?

**Comment by geel9** [comment]
Moving to draft as we are pausing this work

**Comment by tmarkovski** [comment]
Bump @geel9 @fundthmcalculus . We still want this in or can it be closed?

**Pull request closed without merging**